sources/web/datalab/polymer/components/item-list/item-list.ts

```diff
@@ -54,6 +57,16 @@ class ItemListElement extends Polymer.Element {
    */
   public columns: Array<string>;
 
+  /**
+   * Whether to hide the header row
+   */
+  public noHeader: boolean;
+
+  /**
+   * Whether to disable item selection
+   */
+  public noSelection: boolean;
```

Comment:
Same discussion as above, I'll rename to `disable-selection` for now, and we can discuss this in person if you have strong feelings.